schemas/core/components/terms.json

```diff
@@ -64,6 +64,10 @@
     "cancellation": {
       "type": "object",
       "properties": {
+        "cancellationForm": {
```

Comment:
Probably better name, would be `cancellationFormInput` or simply `cancellationReason` (property name should rather be about what the value presents, and not necessarily be influecned via what mean we gather it)

Reply:
this is actually a form url.
There's neither input to this field nor reason 😳

Reply:
this is used when there cannot be a mean to cancel a ticket programmatically (will involves irl actions)

Reply:
Ok, so why it's of type `string` and not `url` ?

Also following HTML terminology name then should be more as `cancallationFormActionUrl`

Reply:
there's no type `url` in json schema afaik
I can change the name to `cancallationFormActionUrl`

Reply:
> there's no type url in json schema afaik

What about
https://github.com/maasglobal/maas-schemas/blob/a17b5e0c6e133f3648009c37684ec2ee170729f1/schemas/core/components/units.json#L10-L14 ? Can't we use that for some reason?

Reply:
Ah can use that ofc. I forgot we have it